quickstart/docker/image/add-router.sh

```diff
@@ -0,0 +1,182 @@
+else
+   er_name="none"
+fi
+
+router_ip=$(ip addr show ${lan_port} | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
+
+echo Use ${router_ip} as internal IP
+
+cat <<EOT >config.yml
+v: 3
+
+identity:
+   cert: "certs/identity.cert.pem"
+   server_cert: "certs/internal.chain.cert.pem"
+   key: "certs/internal.key.pem"
+   ca: "certs/intermediate-chain.pem"
+
+edge:
+  csr:
+    country: US
+    locality: Charlotte
+    organization: Netfoundry
+    organizationalUnit: ADV-DEV
+    #province: NC
+    sans:
+      dns:
+        - "localhost"
+      ip:
+        - "127.0.0.1"
+        - "router_ip"
+        - "external_router_ip"
+
+ctrl:
+  endpoint: tls:edgecontroller:ctrl_port
+
+link:
+  dialers:
+    - binding: transport
+
+listeners:
+  - binding: edge
+    address: tls:0.0.0.0:rport
+    options:
+      advertise: advertise_ip:rport
+      maxQueuedConnects:      50
+      maxOutstandingConnects: 100
+      connectTimeoutMs:       3000
+  - binding: tunnel
+    options:
+      svcPollRate: 15s
+      resolver: udp://router_ip:53
+      dnsSvcIpRange: 100.64.0.1/10
+      lanIf: lan_port
+EOT
+
+#create path for router binaries, certs and congig
+mkdir -p certs
+
+#update config template with external ip of new router
+if [[ "${ZITI_EDGE_ROUTER_IP_OVERRIDE}" != "" ]]; then
+   echo ${ZITI_EDGE_ROUTER_IP_OVERRIDE}
+   # user specified external ip for router, use that for advertisement
+   sed -i "s/external_router_ip/${ZITI_EDGE_ROUTER_IP_OVERRIDE}/" config.yml
+   sed -i "s/advertise_ip/${ZITI_EDGE_ROUTER_IP_OVERRIDE}/" config.yml
+else
+   sed -i "s/advertise_ip/${router_ip}/" config.yml
+   sed -i "s/- \"external_router_ip\"//" config.yml
+fi
+
+#update config template with private ip of new router
+sed -i "s/router_ip/$router_ip/" config.yml
+
+#update config template with public ip of controller
+sed -i "s/edgecontroller/$controller_ip/" config.yml
+
+#update config template with edge listening port of new router
+sed -i "s/rport/$rport/" config.yml
+
+#update config template with controller fabric listening port
+sed -i "s/ctrl_port/$mgmt_port/" config.yml
+
+#update config template with name of lan interface for new router
+sed -i "s/lan_port/$lan_port/" config.yml
+
+if [ -f "ziti/ziti" ]
+then
+   echo "ziti binary found skipping download!"
+else
+   zitilatest=$(curl -s https://api.github.com/repos/openziti/ziti/releases/latest) 
+   version=$(echo "${zitilatest}" | tr '\r\n' ' ' | jq -r '.tag_name' |tr -d v)
+      #download ziti
+   echo ''
+   echo ${BBlue}curl -OL https://github.com/openziti/ziti/releases/download/v${version}/ziti-linux-amd64-${version}.tar.gz${NC}
+   curl -OL https://github.com/openziti/ziti/releases/download/v${version}/ziti-linux-amd64-${version}.tar.gz
+   #untar ziti
+   echo ''
+   echo ${BBlue}tar zxvf ziti-linux-amd64-${version}.tar.gz${NC}
+   tar zxvf ziti-linux-amd64-${version}.tar.gz
+fi
+
+#login to ziti cli
+ziti/ziti edge login "${controller_ip}:${ctrl_port}" -u "admin" -p "admin" -y
```

Comment:
`ziti-cli-functions.sh`, or the .env file after running the quickstart adds a `zitiLogin` alias that works with the quickstarts. i think it's worthwhile to look into that and how it works. it sources an env file and uses the user/pwd specified in there too. if we ever randomize the user/pwd or let the user supply it during the quickstart, you'll be more future proof i think

Reply:
Will also try to work that in there.